src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -211,7 +211,9 @@ HttpStatusCode mongoSubscribeContext
     // 3. Create Subscription for the cache
     if (patternBased && onchange)
     {
-      subCache->insert(new orion::Subscription(tenant, servicePath, requestP, oid.toString(), expiration, notifyFormat));
+      std::string oidString = oid.toString();
+
+      subCache->insert(new orion::Subscription(tenant, servicePath, requestP, oidString, expiration, notifyFormat));
```

Comment:
This is a leak fix, believe it or not ...
When the scope ends, "std::string oidString" on the stack is destroyed.
If I do as before, the oid.toString() creates a std::string and it is lost.
It took me a while to fix this one ...

About style, I try to maintain an empty line between variable-declarations and the rest of the code